packages/gasket-cli/src/scaffold/fetcher.js

```diff
@@ -174,7 +170,8 @@ module.exports = class PackageFetcher {
     return {
       then: (fulfill, reject) => {
         const logOpts = { tarball, dir };
-        const readableStream = fs.createReadStream(tarball).once('error', this._logError(`fs.createReadStream`, logOpts));
+        const fd = open(tarball);
```

Comment:
Don't you need to await this?
```suggestion
        const fd = await open(tarball);
```